cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -13,17 +13,46 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
   override def doc: Document = new DocBuilder {
     doc("Security of Administration", "administration-security-administration")
     database("system")
+    initQueries(
+      "CREATE USER jake SET PASSWORD 'abc123' CHANGE NOT REQUIRED SET STATUS ACTIVE",
+      "CREATE ROLE regularUsers",
+      "CREATE ROLE noAccessUsers",
+      "GRANT ROLE regularUsers TO jake",
+      "DENY ACCESS ON DATABASE neo4j TO noAccessUsers"
+    )
     synopsis("This section explains how to use Cypher to manage Neo4j administrative rights.")
-    section("The 'admin' role", "administration-security-administration-admin") {
-      p("")
+    section("The 'admin' role", "administration-security-administration-introduction") {
+      p("include::admin-role-introduction.asciidoc[]")
+      p("image::grant-privileges-database.png[title=\"GRANT and DENY Syntax\"]")
     }
     section("Database administration", "administration-security-administration-database-privileges") {
-      p("")
+      synopsis("This section explains how to use Cypher to manage privileges for Neo4j database administrative rights.")
```

Comment:
Should this section also contain all other privileges (like start and stop database)? right now it looks like its only contain access